docker/default.env

```diff
@@ -0,0 +1,41 @@
+# Default configuration for Docker containers.
+#
+# To modify, copy values over to ".env" file.
+# Values in ".env" file will override values
+# in "default.env".
+#
+# Values passed via command-line arguments take precedence over .env files:
+# $ WP_DOMAIN=example.com yarn docker:up
+#
+# Note that there is no special handling of quotation marks.
+# This means that they are part of the value.
+#
+# Note that these variables are not available in docker-compose.yml
+# Variables show up defined inside containers only.
+
+# WordPress
+WP_DOMAIN=localhost
+WP_ADMIN_USER=wordpress
+WP_ADMIN_EMAIL=wordpress@example.com
+WP_ADMIN_PASSWORD=wordpress
+WP_TITLE=HelloWord
+
+# Database
+MYSQL_HOST=db:3306
+MYSQL_DATABASE=wordpress
+MYSQL_USER=wordpress
+MYSQL_PASSWORD=wordpress
+MYSQL_ROOT_PASSWORD=wordpress
+
+# SFTP container users (user:pass:UID)
+#
+# Set UID/GID manually for your users if you want them to make changes to
+# your mounted volumes with permissions matching your host filesystem.
+#
+# Define multiple users separated by space
+#
+# Read more: https://github.com/atmoz/sftp
+SFTP_USERS=wordpress:wordpress:1001
```

Comment:
I realised since SFTP container expects this username in [mounted paths](https://github.com/Automattic/jetpack/blob/27578ec36ed24e8510ee445f2e46bbe215948e9b/docker/docker-compose.yml#L42) (`/home/USERNAME/*`), we cannot let users modify this username unless I figure some way to make it dynamic.

Reply:
(Just adding a note and asking people not to modify username would be fine for now)